ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-candidate.yaml

```diff
@@ -35,7 +35,7 @@ tests:
       SLEEP_DURATION: "0"
       TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
       TEST_RELEASE_TYPE: Pre-GA
-      TEST_SCENARIOS: C00102
+      TEST_SCENARIOS: sig-kata
```

Comment:
This needs to be more restrictive
There are a number of tests in sig-kata that don't play well with others.
I use
`sig-kata.*Kata Author'`